Source/Core/Core/State.cpp

```diff
@@ -74,7 +74,7 @@ static Common::Event g_compressAndDumpStateSyncEvent;
 static std::thread g_save_thread;
 
 // Don't forget to increase this after doing changes on the savestate system
-static const u32 STATE_VERSION = 99;  // Last changed in PR 6020
+static const u32 STATE_VERSION = 100;  // Last changed in PR 7589
```

Comment:
Incorrect PR number.

Reply:
Yeah. That was the old PR. I'll change it now that this PR exists. :P